restapi.go

```diff
@@ -320,6 +320,23 @@ func (s *Session) UserSettings() (st *Settings, err error) {
 	return
 }
 
+// UserUpdateStatus update the user status
+// status   : The new status (Actual valid status are 'online','idle','dnd','invisible')
+func (s *Session) UserUpdateStatus(status string) (st *Settings, err error) {
```

Comment:
The problem is actually you have 5 status.
"online","idle","dnd","invisible","offline".
However the Status can be set by the user as "online","idle","dnd" or "invisible"
And the library can see only Users as "online","idle","dnd" or "offline"
So even if there is a Status type the user can send a status bad type ("offline")
I will do that and make a verification in the function.